lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -1,38 +1,56 @@
 .adf {
     &-metadata-properties {
-        .mat-expansion-panel-header.mat-expanded:hover,
-        .mat-expansion-panel-header.mat-expanded:focus {
-            background: var(--adf-theme-background-hover-color);
-        }
-
         mat-expansion-panel-header {
-            height: 64px;
+            height: 56px;
 
             .adf-metadata-properties-title {
-                font-weight: normal;
+                font-weight: 700;
                 font-size: 15px;
+                padding-left: 12px;
             }
         }
 
         .mat-expansion-panel:not([class*='mat-elevation-z']) {
             box-shadow: none;
         }
 
-        .adf-metadata-properties-tag {
-            height: 40px;
+        .adf-mat-divider {
+            margin-left: -24px;
+            margin-right: -24px;
+        }
+
+        .adf-edit-icon-buttons {
+            color: var(--adf-theme-foreground-text-color-054);
+        }
+
+        .adf-metadata-properties-panel-content {
             display: flex;
             align-items: center;
-            margin-top: -14px;
-            margin-bottom: 1em;
+            flex: 1;
+        }
 
-            &:first-of-type {
-                margin-top: 5px;
-            }
+        .adf-metadata-properties-tag {
+            height: 32px;
+            display: inline-flex;
+            align-items: center;
+            border-radius: 16px;
+            width: fit-content;
+            background: var(--theme-metadata-buttons-background-color);
+            margin-top: 12px;
+            padding: 6px 12px;
+            justify-content: center;
+            margin-left: 8px;
```

Comment:
This makes firstly added tag misaligned with input field, can we fix that? We can apply margin for each element except the first one

Reply:
Done.